src/sentry/tasks/post_process.py

```diff
@@ -1189,10 +1189,17 @@ def process_service_hooks(job: PostProcessJob) -> None:
         if has_alert:
             allowed_events.add("event.alert")
 
-        if allowed_events:
-            for servicehook_id, events in _get_service_hooks(project_id=event.project_id):
-                if any(e in allowed_events for e in events):
-                    process_service_hook.delay(servicehook_id=servicehook_id, event=event)
+        # TODO: when we have multiple service hook versions, this will need to change.
+        payload = json.dumps(get_payload_v0(event))
+
+        for servicehook_id, events in _get_service_hooks(project_id=event.project_id):
+            if any(e in allowed_events for e in events):
+                process_service_hook.delay(
+                    servicehook_id=servicehook_id,
+                    project_id=event.project_id,
+                    event=event,
+                    payload=payload,
+                )
```

Comment:
Because `process_service_hook` already supports `kwargs` we can provide the new args at the same time as expanding the signature.